### DISCUSSION TITLE
For viewer is there no way default view as TV mode?

### DISCUSSION THREAD

**Discussion by type-kimchi** [grafana/grafana#57050] [Dashboards, Panels & Visualizations]
In my opinion there is no need to sidebar for viewer.

When i add url &kiosk=tv it is not work for default view..

I 'd like to navigate for TV mode only for viewers.

Could you please advise it or work around ?

thanks, Best regards.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
